packages/core/src/js/feedback/FeedbackWidget.tsx

```diff
@@ -406,6 +406,18 @@ export class FeedbackWidget extends React.Component<FeedbackWidgetProps, Feedbac
     return this.state.filename !== undefined && this.state.attachment !== undefined && this.state.attachmentUri !== undefined;
   }
 
+  private _getUser = (): User | undefined => {
+    const currentUser = getCurrentScope().getUser();
+    if (currentUser) {
+      return currentUser;
+    }
+    const isolationUser = getIsolationScope().getUser();
+    if (isolationUser) {
+      return isolationUser;
+    }
```

Comment:
i: Most people set the user by using Sentry.setUser, with that we could invert the `currentScope` with `isolationScope` so it return in most of the cases on first try.

```suggestion
    const isolationUser = getIsolationScope().getUser();
    if (isolationUser {
      return isolationUser
    }
    const currentUser = getCurrentScope().getUser();
    if (currentUser) {
      return currentUser
    }
```

Reply:
we can skip the suggeston.
To me it sounds like, on react native context, where you'll only have one user, Sentry.SetUser should use the global scope instead of an isolation scope.

Maybe it could be a nice nit that we could change on V7 if others agree.